tests/e2e/lib/init/init.go

```diff
@@ -0,0 +1,104 @@
+package init
+
+import (
+	"flag"
+	"os"
+	"time"
+)
+
+// Common vars obtained from flags passed in ginkgo.
+var credFile, namespace, credSecretRef, instanceName, provider, ci_cred_file, settings, artifact_dir, oc_cli string
```

Comment:
There has to be a better way to do this stuff than global variables IMO. I also am very concerned with the package name.

Reply:
re: package variable. only accessed via getters. these are string and has no pointers.